content/stylesheets/pages/_post.scss

```diff
@@ -107,149 +107,50 @@
     display: flex;
     margin-left: auto;
     position: relative;
-    height: 2rem;
+    height: 2.5rem;
 
-    .share-toggle-button {
-      position: relative;
-      display: inline-flex;
-      align-items: center;
-      justify-content: center;
-      gap: 0.5rem;
-      padding: 0 1rem;
-      font-weight: 600;
-      font-size: 0.85rem;
-      cursor: pointer;
+    .app-button--primary {
       z-index: 10;
-      color: hsl(203, 89%, 47.5%);
-      background: hsla(184, 100%, 35%, 0.106);
-      border: 1px solid hsla(184, 100%, 20%, 0);;
-      border-radius: 0.6rem;
-      white-space: nowrap;
       transition: all 0.2s ease;
```

Comment:
current transition doesn't 'feel' good to me (too fast?), not needed if we remove the button